assets/scss/components/_preview-cart.scss

```
@@ -0,0 +1,92 @@
+.c-cart-preview__container {
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+}
+
+.c-cart-preview {
+    display: none;
+    position: fixed;
+    right: 0;
+    top: 0;
+    max-width: rem-calc(390px);
+    width: 100%;
+    height: 100vh;
+    background-color: $color-white;
+    z-index: $z-index-very-high;
+    padding: rem-calc(24px);
+    padding-bottom: 0;
+
+    &.loaded {
```

By default, the element should be styled as shown, but you can hide it via the `is-hidden` classname. You may feel like "what's the difference?" but having a consistent approach through projects will make your life (and any developers in a future) easier :)